extensions/duckduckgo-image-search/src/search-image.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useRef, useState } from "react";
+import { Action, ActionPanel, closeMainWindow, Grid, Icon, Image, PopToRootType, showHUD } from "@raycast/api";
+
+import { useCachedPromise } from "@raycast/utils";
+import { PaginationOptions } from "@raycast/utils/dist/types";
+import { copyImageToClipboard, pasteImage, searchImage } from "../utils/helpers";
+import { ImageLayout, ImageLayouts } from "../utils/consts";
+import { DuckDuckGoImage } from "../utils/search";
+
+function ActionsPanel({ item }: { item: DuckDuckGoImage }) {
+  return (
+    <ActionPanel>
+      <Action.OpenInBrowser title="Open Image in Browser" url={item.image} />
+      <Action
+        title="Paste Image"
+        shortcut={{
+          modifiers: ["cmd", "shift"],
+          key: "enter",
+        }}
+        icon={Icon.Clipboard}
+        onAction={() =>
+          pasteImage(item).then(async () => {
+            await closeMainWindow({
+              clearRootSearch: true,
+            });
+          })
+        }
+      />
+      <Action.OpenInBrowser
+        title="Open Site in Browser"
+        url={item.url}
+        shortcut={{
+          modifiers: ["shift"],
+          key: "enter",
+        }}
+      />
+      <Action
+        title="Copy Image"
+        shortcut={{
+          modifiers: ["cmd"],
```

Comment:
This shortcut makes it impossible to copy the search query to clipboard since it overrides CMD + C, WDYT about changing it to require a "shift" too? Just a thought.

Reply:
Thank you for the information; I was unaware of this behavior! Changed to ⌘⇧C